source/_components/sensor.trafikverket_weatherstation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Trafikverket WeatherStation"
+description: "Instructions how to integrate Trafikverket WeatherStation within Home Assistant."
+date: 2018-02-01 12:06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: trafikverket.png
+ha_category: Sensor
+ha_release: 0.63.0
+---
+
+Showing weather information for air and road temperature provided by Trafikverket in Sweden. 
```

Comment:
Trafikverket could be a link to their homepage here.

Reply:
Updated, please review.